makeWorkspace/make_ws.py

```diff
@@ -41,90 +120,25 @@ def main():
     os.makedirs(outdir)
 
   fin = ROOT.TFile(args.file,'READ')
-  f_jes = ROOT.TFile("sys/shape_jes_uncs_smooth.root")
+  # JES shape files for each category
```

Comment:
While we are already cleaning up: Can we refactor this dictionary + the for loop that figures out which file to use based on the category name into a function? 
```python
def get_jes_file(category):
   # code to figure out right file goes here
   ...
   return TFile("/path/to/right/file")
```

Reply:
Sure, will do this